src/sentry/api/event_search.py

```diff
@@ -2042,6 +2043,19 @@ def result_type_fn(function_arguments, parameter_values):
             transform="uniqIf(user, greater(duration, {tolerated:g}))",
             default_result_type="number",
         ),
+        Function(
+            "misery",
+            required_args=[NumberRange("satisfaction", 0, None)],
+            calculated_args=[{"name": "tolerated", "fn": lambda args: args["satisfaction"] * 4.0}],
+            # To correct for sensitivity to low counts, User Misery is modeled as a Beta Distribution Function.
+            # With prior expectations, we have picked the expected mean user misery to be 0.05 and variance
+            # to be 0.0004. This allows us to calculate the alpha (5.8875) and beta (111.8625) parameters,
+            # with the user misery being adjusted for each fast/slow unique transaction. See:
+            # https://stats.stackexchange.com/questions/47771/what-is-the-intuition-behind-beta-distribution
+            # for an intuitive explanation of the Beta Distribution Function.
+            transform="divide(plus(uniqIf(user, greater(duration, {tolerated:g})), 5.8875), plus(uniq(user), 117.75))",
```

Comment:
Looks like `snuba` is adding a `ifNull` around the denominator (`uniq(user)`) for us but not the numerator (`uniqIf(user, duration > 1200)`) so if there are no users, this results in a `null` being returned breaking the frontend since it's expecting a non nullable number.